server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessageListMethodTest.java

```diff
@@ -1332,6 +1332,256 @@ public void getMessageListShouldComputeTextBodyWhenNoTextBodyButHtmlBody() throw
         ;
     }
 
+    @Test
+    public void getMessageListHasKeywordFilterShouldWork() throws Exception {
```

Comment:
please put a better description for each method containing "ShouldWork". We try to not name test method with "ShouldWork" as it does not expose the intent of the code.